test/OpenTelemetry.Instrumentation.Http.Tests/HttpClientTests.Basic.cs

```diff
@@ -278,7 +278,7 @@ public async Task InjectsHeadersAsync_CustomFormat()
         }));
     }
 
-    [Fact]
+    [Fact(Skip = "Removed for stable release of http instrumentation")]
```

Comment:
lets add a link to a tracking gh issue.

Reply:
Yes, that is my TODO on PR desc :D

Reply:
Also add in PR desc and/or changelog the full impact of this PR.

Reply:
updated